reports/2022.html

```diff
@@ -199,51 +203,157 @@ <h3>Links</h3>
           <dt>Previous WCCG Report(s)</dt>
           <dd><a href="https://w3c.github.io/webcomponents-cg/index.html#form-associated-custom-elements">2021</a></dd>
           <dt>GitHub issues:</dt>
-          <dd>---</dd>
+          <dd><a href="https://github.com/whatwg/html/issues">https://github.com/whatwg/html/issues</a></dd>
```

Comment:
Do we think we need / want a dedicated link to the issue instead of a repo?  Just curious if that would be helpful from reader perspective?  Or we also add this link in here at the top for reference?
https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-face-example

Reply:
Yeah, I wrestled with that. Unfortunately there seem to be a number of separate issues since this really is a collection of features/APIs.

Reply:
I’d say a perponderance of links are better than a dearth. It could allow us to draw a fine line as to what does 2022 support look like down the line. However, I don’t see it specifically blocking this pass at the data!